chromium_src/chrome/browser/printing/print_view_manager_base.cc

```diff
@@ -377,9 +378,14 @@ void PrintViewManagerBase::DisconnectFromCurrentPrintJob() {
 }
 
 void PrintViewManagerBase::PrintingDone(bool success) {
-  if (!print_job_.get())
-    return;
-  Send(new PrintMsg_PrintingDone(routing_id(), success));
+  bool is_success = false;
+  if (print_job_.get()) {
+    is_success = success;
+    Send(new PrintMsg_PrintingDone(routing_id(), success));
+  }
+  if (callback_ptr.get()) {
+    callback_ptr->Run(is_success);
```

Comment:
The `is_success` variable seems unnecessary, you can just call `callback_ptr->Run(success && print_job_)`.